crypto/kdf/tls1_prf.c

```diff
@@ -184,7 +184,8 @@ static int tls1_prf_P_hash(const EVP_MD *md,
     int ret = 0;
 
     chunk = EVP_MD_size(md);
-    OPENSSL_assert(chunk >= 0);
+    if (!ossl_assert(chunk > 0))
+        goto err;
```

Comment:
One could say that last previous argument applies even here, but EVP_MD_size actually can return value other than md->md_size. One can probably argue that the way EVP_MD_size is implemented is not directly justified, i.e. it should not tolerate md being NULL. But that's probably topic for another discussion[?]...